app/role_data.go

```diff
@@ -174,6 +174,7 @@ func addRoles(management *config.ManagementContext) (string, error) {
 		addRule().apiGroups("management.cattle.io").resources("catalogtemplates").verbs("*").
 		addRule().apiGroups("management.cattle.io").resources("catalogtemplateversions").verbs("*").
 		addRule().apiGroups("monitoring.cattle.io").resources("prometheus").verbs("view").
+		addRule().apiGroups("monitoring.coreos.com").resources("prometheuses", "prometheusRules", "serviceMonitors", "alertmanagers").verbs("*").
```

Comment:
why are these additional permissions needed?

Reply:
This is the privileges of project owner. As the project member have the full privileges to monitoring crds(`prometheus`, `prometheusRules` and `serviceMonitor`), the project owner should have them too.